esmvalcore/preprocessor/_derive/co2s.py

```diff
@@ -15,6 +15,8 @@ def _get_first_unmasked_data(array, axis):
     indices = da.meshgrid(
         *[da.arange(array.shape[i]) for i in range(array.ndim) if i != axis],
         indexing='ij')
+    # numpy>=2.0 indices becomes a tuple
```

Comment:
Remove comment? I don't think this is useful

Reply:
done in 85b2aa4f6523a4ec1d2eb2e2d00db12d17f6b405